tests/data/string_prefixes.py

```diff
@@ -1,18 +1,24 @@
 #!/usr/bin/env python3.6
 
-name = R"Łukasz"
-F"hello {name}"
-B"hello"
-r"hello"
-fR"hello"
+name = "Łukasz"
+(f"hello {name}", F"hello {name}")
+(b"", B"")
+(u"", U"")
+(r"", R"")
+
+(rf"", fr"", Rf"", fR"", rF"", Fr"", RF"", FR"")
+(rb"", br"", Rb"", bR"", rB"", Br"", RB"", BR"")
 
 # output
 
 
 #!/usr/bin/env python3.6
```

Comment:
NIT: I feel we should just go to python3 here
```suggestion
#!/usr/bin/env python3
```

Reply:
Consider it done!